src/Http/Controllers/AppBaseController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace EscolaLms\Consultations\Http\Controllers;
+
+use EscolaLms\Core\Http\Controllers\EscolaLmsBaseController;
+
+/**
+ * SWAGGER_VERSION
+ * This class should be parent class for other API controllers
+ * Class AppBaseController
+ */
+class AppBaseController extends EscolaLmsBaseController
+{
+    public function sendDataError($error, $data, $code = 422)
```

Comment:
Are you using this function anywhere? If not, then your controllers should extend EscolaLmsBaseController directly

Reply:
Removed this controller